addons/variable-manager/userscript.js

```diff
@@ -0,0 +1,168 @@
+export default async function ({ addon, global, console, msg }) {
+  const vm = addon.tab.traps.vm;
+  window.vm = vm; // for debugging if i forget to commehnt this out plz yel at me thanks
+
+  /*   let globals = vm.runtime.getTargetForStage().variables
+    let local = vm.runtime.getEditingTarget().variables */
+  addon.tab.redux.initialize();
+  while (true) {
+    let tabs = await addon.tab.waitForElement("[class^='react-tabs_react-tabs__tab-list']", {
+      markAsSeen: true,
+    });
+    if (tabs) {
+      let soundTab = tabs.children[2];
+      let contentArea = document.querySelector(".gui_tabs_AgmuP");
+
+      let manager = document.createElement("div");
+      manager.className = "asset-panel_wrapper_366X0 sa-var-manager"; // TODO: how do i make this work even if css rebuilt and the random thing changes
```

Comment:
Use `addon.tab.scratchClass`.

Reply:
thanks

Reply:
You don't even need to use `classList`, because `scratchClass` supports multiple arguments: `addon.tab.scratchClass("gui_tab", "asset-panel_wrapper", {others: ["sa-var-manager"]})`.

Reply:
oh wow! i'll change it then.